neo4j/README.md

```diff
@@ -2,11 +2,20 @@
 
 ## Overview
 
-This check monitors [neo4j][1] through the Datadog Agent.
-Please verify the metrics and checks that are submitted through this agent. Since Neo4j 4.0 and onward supports multiple databases, some metrics and checks are no longer published.
+The Neo4j integration for Datadog enables data collection and alerting on a [Neo4j][1] metrics available at the Prometheus endpoint, using the Datadog platform.
```

Comment:
The overview should include: 
- What Neo4j does in 1-2 sentences
- The value of this integration (this is mentioned but would be helpful to know the value of collection and alerting on neo4j)
- What data is collected